app/utils/redirect-auth.js

```diff
@@ -0,0 +1,10 @@
+import { AUTH } from '../constants/urls';
+
+export default function () {
+  let authUrl = AUTH.GITHUB_SIGN_IN;
+  if (typeof window !== 'undefined') {
+    const separator = authUrl.includes('?') ? '&' : '?';
+    authUrl = `${authUrl}${separator}redirectURL=${encodeURIComponent(window.location.href)}`;
+  }
+  window.open(authUrl, '_self');
+}
```

Comment:
- change this function to a named export.
- can we use `UrlSearchParams` to handle search params instead of doing `authUrl.includes('?') ? '&' : '?';`?
- can we use `window.location.href` instead of window.open?